desci-server/src/services/data/externalUrlProcessing.ts

```diff
@@ -155,7 +157,7 @@ export async function processExternalUrlDataToIpfs({
     // Pin new files, add draftNodeTree entries
     if (externalUrlFiles?.length) {
       // External URL non-repo
-      pinResult = await pinNewFiles(externalUrlFiles, true);
```

Comment:
lol I'm dying to refactor this function into like 5 different ones for all these cases :joy: